ccd-definition/State.json

```diff
@@ -67,6 +67,27 @@
     "ID": "HEARING_READINESS",
     "Name": "Hearing readiness",
     "Description": "Claim is readied for hearing",
-    "DisplayOrder": 9
+    "DisplayOrder": 10
+  },
```

Comment:
@pliao-hmcts  please add a comment in the file below that these are future states not yet introduced so that its more clear thanks